enhancements/network/service-egress-traffic-steering.md

```diff
@@ -43,7 +46,7 @@ Therefore they require that when one of these pod applications initiate traffic
 
 - Announcing the service externally (for service ingress traffic) with OVN-Kubernetes - this part should be handled by a LoadBalancer provider (like MetalLB) as explained later.
 
-- Using the service's ingress IP for pod to pod traffic.
+- Using the service's ingress IP for pod to pod traffic, pod to node traffic, pod to service traffic.
```

Comment:
I thought @tssurya fix was that pod to node does not use the ingress IP and just uses the node ip?

Reply:
correct uses nodeIP.

Reply:
pod to other node is considered in-cluster traffic for egressIPs and egressSVC. It's not "egress" to cluster.